### ISSUE TITLE
Publish Java Libraries to Maven Central Repository

### ISSUE THREAD

**Issue by howard-abrams** [Microsoft/malmo#469]
I've submitted a pull request to deeplearning4j to add built-in Malmo support. The code is straight forward, but depends on "MalmoJavaJar.jar" and "libMalmoJava.jnilib".

Because deeplearning4j is in the maven central repo, it causes difficulties to have it depend on something that is a separate, manual, download.  Ideally, the Malmo team would publish the java artifacts into the maven central repository with each release.

Is that something the team is willing/interested in doing?

**Comment by DaveyBiggers** [comment]
Thanks for bringing Malmo to deeplearning4j! Interesting idea, we could look into publishing the java artifacts, though Malmo itself would introduce manual dependencies - ffmpeg, codesynthesis etc...

**Comment by howard-abrams** [comment]
Yes, I think at this point we're only talking about the java artifacts - the jar and the various platform-specific JNI libraries.  That would resolve buildtime dependencies and we'd have to doc the remaining runtime requirements.

**Comment by DaveyBiggers** [comment]
We're keen to support deeplearning4j if we can, so we'll take a look at this. Thanks again for the suggestion.

**Comment by agibsonccc** [comment]
Hey guys - just a brief comment here. I was excited to see this actually. I just can't expect our users to build from source. We deploy all of our artifacts to maven central because it's what people expect.
We ourselves also use it for stable releases. If you need any help please feel free to reach ou.

**Comment by howard-abrams** [comment]
In the meantime, following sonatype's policy for 3rd party artifacts, I've had them post 0.30.0 to maven central. If/when the malmo team is ready to publish automatically, you guys can do so.
